hypervisor/arch/x86/guest/vmcs.c

```diff
@@ -360,7 +360,7 @@ static void init_exec_ctrl(struct acrn_vcpu *vcpu)
 	 * TODO: introduce API to make this data driven based
 	 * on VMX_EPT_VPID_CAP
 	 */
-	value64 = hva2hpa(vm->arch_vm.nworld_eptp) | (3UL << 3U) | 6UL;
+	value64 = hva2hpa(vm->arch_vm.nworld_eptp) | (3UL << 3U) | 6UL | (1UL << 6U);
```

Comment:
Current we didn't check whether accessed and dirty flags for EPT are supported in IA32_VMX_EPT_VPID_CAP MSR.

Reply:
No, we didn't. We should do that just like the TODO says when push the patch to master.